Dockerfile

```diff
@@ -8,15 +8,15 @@ LABEL org.opencontainers.image.revision="v0.0.1"
 
 ARG DEBIAN_FRONTEND=noninteractive
 ARG JAVA_VERSION=17
-ARG PRINCE_VERSION=15.1
+ARG PRINCE_VERSION=15.3
 ARG SAXON_VERSION=SaxonHE12-4
```

Comment:
```suggestion
ARG SAXON_VERSION=SaxonHE12-5
```

Reply:
thx for spotting, we’ll postpone this a bit but won’t forget, cf. #44 ;-)